### PR TITLE
Added issue templates

### PR DESCRIPTION
**Pull request by kanak22**
## Related Issue 
Added issue templates

### Describe the changes you've made
I have added issue templates for bug reports, feature requests and a custom issue template.

## Type of change

What sort of change have you made:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Enhancement in the existing features
- [x] Other

## Mention any unusual behaviour of your code (Write NA if not)
NA

## Checklist:

- [x] My code follows the guidelines of this project.
- [x] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly whereever it was hard to understand.
- [x] My changes generate no new warnings.
- [x] I have verified/tested my code by running it locally.

### CODE REVIEW

**Comment by Namanl2001** [comment]
Please resolve merge conflict in `.github/pull_request_template.md`

**Comment by kanak22** [comment]
I added it. I guess its not a conflict still. If it is so can u help me out as I m a beginner 😐

**Comment by Namanl2001** [comment]
Delete all the changes in green in `.github/pull_request_template.md` shown in https://github.com/Namanl2001/MERN-Gurujii-dev/pull/44/files.

File: .github/ISSUE_TEMPLATE/custom.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+name: Custom issue template
+about: Describe this issue template's purpose here.
```
Comment (Namanl2001) [inline .github/ISSUE_TEMPLATE/custom.md:3]: `Neither a bug nor a feature`

**Review by Namanl2001 — CHANGES_REQUESTED** [review]
LGTM! with some small nits.

File: .github/ISSUE_TEMPLATE/custom.md (hunk 1 above)
Comment (kanak22) [reply .github/ISSUE_TEMPLATE/custom.md]: did it

File: .github/ISSUE_TEMPLATE/custom.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+---
+name: Custom issue template
```
[the remark this answers is not in the document]
Comment (kanak22) [reply .github/ISSUE_TEMPLATE/custom.md]: did it

File: .github/ISSUE_TEMPLATE/bug-report.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
+---
+name: Bug report
```
[the remark this answers is not in the document]
Comment (kanak22) [reply .github/ISSUE_TEMPLATE/bug-report.md]: did it

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
+---
+name: Feature request
```
[the remark this answers is not in the document]
Comment (kanak22) [reply .github/ISSUE_TEMPLATE/feature_request.md]: did it

**Review by Namanl2001 — APPROVED** [review]
Great work! @kanak22 will merge on green. Thanks!

**Comment by kanak22** [comment]
thankyou!! 🤗

**Pull request merged**